sw/device/lib/crypto/drivers/otbn.c

```diff
@@ -35,10 +35,41 @@ ASSERT_ERR_BIT_MATCH(kOtbnErrBitsLifecycleEscalation,
 ASSERT_ERR_BIT_MATCH(kOtbnErrBitsFatalSoftware,
                      OTBN_ERR_BITS_FATAL_SOFTWARE_BIT);
 
-const size_t kOtbnDMemSizeBytes = OTBN_DMEM_SIZE_BYTES;
-const size_t kOtbnIMemSizeBytes = OTBN_IMEM_SIZE_BYTES;
+enum {
+  /**
+   * Base address for OTBN.
+   */
+  kBase = TOP_EARLGREY_OTBN_BASE_ADDR,
+  /**
+   * DMEM size in bytes.
+   */
+  kOtbnDMemSizeBytes = OTBN_DMEM_SIZE_BYTES,
+  /**
+   * IMEM size in bytes.
+   */
+  kOtbnIMemSizeBytes = OTBN_IMEM_SIZE_BYTES,
+};
 
-enum { kBase = TOP_EARLGREY_OTBN_BASE_ADDR };
+/**
+ * OTBN commands
+ */
+typedef enum otbn_cmd {
+  kOtbnCmdExecute = 0xd8,
+  kOtbnCmdSecWipeDmem = 0xc3,
+  kOtbnCmdSecWipeImem = 0x1e,
+} otbn_cmd_t;
+
+/**
+ * OTBN status
+ */
+typedef enum otbn_status {
+  kOtbnStatusIdle = 0x00,
+  kOtbnStatusBusyExecute = 0x01,
+  kOtbnStatusBusySecWipeDmem = 0x02,
+  kOtbnStatusBusySecWipeImem = 0x03,
+  kOtbnStatusBusySecWipeInt = 0x04,
+  kOtbnStatusLocked = 0xFF,
+} otbn_status_t;
```

Comment:
Are there auto-generated macros for these values?

Reply:
Apparently not. Created https://github.com/lowRISC/opentitan/issues/16754 to track this.